### DISCUSSION TITLE
Cables page loader

### DISCUSSION THREAD

**Discussion by visuality** [cables-gl/cables_docs#458] [Answers & Questions] [answered]
How do you turn on the page loader function, I see it on all the content when we preview examples on the Cables, is it customize with my own logo?

**Comment by steam0r** [comment, marked as answer]
check out this op for a simple solution:

https://cables.gl/op/Ops.Html.LoadingIndicator

you can use the information of this op to create a custom indicator:

https://cables.gl/op/Ops.Patch.LoadingStatus_v2

**Comment by visuality** [comment]
Thank you so much for the answer, the page loaders seem a little buggy sometimes loading and sometimes not within the cables editor so since it is for my portfolio website and I did not want to make an error, I integrated another simple css based loader that seems to work fine but I rethought the whole idea of putting something so heavy on my main page. Just wanted to thank you for the answer.
